cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -732,10 +737,9 @@ TEST(Substrait, ExtensionSetFromPlan) {
     EXPECT_EQ(decoded_null_type.id.name, "null");
     EXPECT_EQ(*decoded_null_type.type, NullType());
 
-    EXPECT_OK_AND_ASSIGN(auto decoded_add_func, ext_set.DecodeFunction(42));
-    EXPECT_EQ(decoded_add_func.id.uri, kArrowExtTypesUri);
-    EXPECT_EQ(decoded_add_func.id.name, "add");
-    EXPECT_EQ(decoded_add_func.name, "add");
```

Comment:
did you mean to drop this one?

Reply:
`EXPECT_EQ(decoded_add_func.name, "add");`

Reply:
`DecodeFunction` used to return an `Id` and an Arrow function name.  Returning only an Arrow function name was not sufficient.  Now it only returns an `Id`.  The job was split into `DecodeFunction` (which the extension set provides itself) and `GetSubstraitCallToArrow` (which the extension id registry provides).  So there is no `decoded_add_func.name` any longer.

Reply:
Got it. I missed that.